network/datadog_checks/network/network.py

```diff
@@ -12,12 +12,39 @@
 from collections import defaultdict
 
 import psutil
-from six import PY3, iteritems
+from six import PY3, iteritems, itervalues
+
+from datadog_checks.base.checks import AgentCheck
+from datadog_checks.base.utils.common import pattern_filter
+from datadog_checks.base.utils.platform import Platform
+from datadog_checks.base.utils.subprocess_output import SubprocessOutputEmptyError, get_subprocess_output
+
+try:
+    # Try first the non-C, non-io version on py2. This is the one that will
+    # handle unicode properly, without copy.
+    from StringIO import StringIO
+except ImportError:
+    # We don't have the same thing on py3, so reimplement a small subset for
```

Comment:
Can't we use https://docs.python.org/3/library/io.html#io.StringIO?

`from six import StringIO`

Reply:
Removed.